.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Dataflow_Java11.groovy

```diff
@@ -20,26 +20,40 @@ import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 
 
-PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_Dataflow',
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11',
   'Run Dataflow ValidatesRunner Java 11', 'Google Cloud Dataflow Runner ValidatesRunner Tests On Java 11', this) {
```

Comment:
I suggest use trigger phrase
"Run beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11"
it is same in terms of being descriptive, but much easier to figure out.

Or "Run Google Cloud Dataflow Runner ValidatesRunner Tests On Java 11".

Reply:
I recently changed other ValidatesRunner trigger phrases to be similar to this one, wouldn't it be better to be consistent here?